hibernate-core/src/main/java/org/hibernate/boot/cfgxml/internal/ConfigLoader.java

```diff
@@ -103,7 +105,9 @@ public LoadedConfig loadConfigXmlUrl(URL url) {
 					stream.close();
 				}
 				catch (IOException e) {
-					log.debug( "Unable to close cfg.xml URL stream", e );
+					if (log.isDebugEnabled()) {
+						log.debug( "Unable to close cfg.xml URL stream", e );
+					}
```

Comment:
Hard to see how this is useful, TBH. We're not doing any string manipulation here, and that log only occurs when "something went wrong". I would leave it as it was.